reason-react-native/src/apis/Style.re

```diff
@@ -1,5 +1,25 @@
 type t;
 
+external array: array(t) => t = "%identity";
+external arrayOption: array(option(t)) => t = "%identity";
+/* list works too since RN accept recursive array of styles (ocaml list are recursive arrays in JS)*/
+external list: list(t) => t = "%identity";
+external listOption: list(option(t)) => t = "%identity";
+
+[@bs.val]
+external assign2: (Js.t('a), Js.t('b), Js.t('c)) => Js.t('d) =
+  "Object.assign";
+
+// Escape hatch, in case something is added into RN but unsupported,
+// Useful if you play with fancy platforms
+// Use with caution
+let unsafeAddStyle: (t, Js.t('a)) => t =
+  (style, styles) => {
+    Obj.(assign2(Js.Dict.empty()->magic, style->magic, styles->magic)->magic);
```

Comment:
Slightly pedantic, but we could instead have
```reason
    Obj.(assign2(Js.Obj.empty(), style->magic, styles->magic)->magic);
```
which saves us one piece of `magic`, although JS output does not change even slightly.

Reply:
Agreed

Reply:
I had to change assign2 method so it takes a Js.Dict for the first arg.